inst/www/shared/shiny.css

```diff
@@ -3,12 +3,9 @@ body.disconnected {
   opacity: 0.5;
 }
 
-table.data {
-  width: auto;
-}
-table.data td[align=right] {
-  font-family: monospace;
-  text-align: right;
+.shiny-table th {
+  padding-left: 12px !important;
```

Comment:
Because there is a bunch of Bootstrap padding defaults. If the `!important` is not included, these lines are ignored in the final rendered page...